adding_species_manuscript.tex

```diff
@@ -802,9 +803,9 @@ \section*{Conclusion}\label{conclusion}}
 easy inclusion of new species genomes, and community-maintained accuracy
 and correctness. We additionally briefly describe how to add species to
 the stdpopsim catalog, and how the quality control process for species
-inclusion works. Currently large-scale projects such as the Darwin Tree
-of Life (TODO: CITE) project are generating tens of thousands of genome
-assemblies-- each of these, with some prior knowledge of mutation and
+inclusion works. Currently large-scale efforts such as the Earth Biogenome
+and its affiliated project networks are generating tens of thousands of genome
```

Comment:
I don't know the landscape here - so DTOL is an affiliate project of EBP? (should be "earth biogenome project", btw) Has anyone else actually produced lots of assemblies yet besides DTOL? If not it's worth singling out DTOL here (although mentioning EBP too?).

Reply:
True, DTOL is churning more out at the moment, but my understanding is that others are on that track. I am happy to go either way, but though that it would be good to point the general state of afairs.